scripts/syntactics/index.rst

```diff
@@ -0,0 +1,78 @@
+Deep Biaffine Dependency Parser
+---------------------------------
+
+This package is an implementation of `Deep Biaffine Attention for Neural Dependency Parsing <https://arxiv.org/pdf/1611.01734.pdf>`_ proposed by Dozat and Manning (2016), with SOTA accuracy.
+
+Train
+^^^^^^^^^^^^^
+
+As the Penn Treebank dataset (PTB) is proprietary, we are unable to distribute it.
+If you have a legal copy, please place it in ``tests/data/biaffine/ptb``, use this `pre-processing script <https://github.com/hankcs/TreebankPreprocessing>`_ to convert it into conllx format.
+Then download `glove embeddings <http://nlp.stanford.edu/data/glove.6B.zip>`_ into ``tests/data/biaffine/embedding`` and unzip it.
```

Comment:
I understand that vocab needs to be improved before this model can adopt the existing vocab class. On the other hand, glove embedding in gluonnlp can be used independent of the vocab class, and glove embeddings are supported. Can we adapt that so that there's no need for users to download the embeddings themselves?

Reply:
Yes, I'll use `gluonnlp.embedding.create`. But can I remove the support of custom embeddings, but only accept instances of `gluonnlp.embedding.TokenEmbedding` or their source names?

Reply:
Yes, that's perfectly fine.